index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main id="main">
+        <h1 id="title">Raila Odinga </h1>
+        <p>Studied at Leipzig University</p>
+        <figure id="img-div">
+            <img id="image" src="images/Odinga.jpeg" alt="#" />
+            <figcaption id="img-caption">
+                Raila Odinga was born at the Anglican Church Missionary Society Hospital,
+                in Maseno, to Mary Juma Odinga and Jaramogi Oginga Odinga. His father served as the first Vice President
+                of Kenya under President Jomo Kenyatta. He is a member of the Luo ethnic group.
+            </figcaption>
+        </figure>
+        <section id="tribute-info">
+            <h3 id="headline">The following is Raila Odinga life:</h3>
+            <ul>
+                <li><strong>1945</strong> - Kisumu District, Nyanza Province on 7 January 1945</li>
```

Comment:
- could style the list items on the styles sheet and avoid using strong tag , this gives you more options.
- please implement this in all areas where the strong tag was used , line 68, 104 and others.